config/apiserver.yaml

```diff
@@ -0,0 +1,189 @@
+
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.federation.k8s.io
+  labels:
+    api: federation
+    apiserver: "true"
+spec:
+  version: v1alpha1
+  group: federation.k8s.io
+  groupPriorityMinimum: 2000
+  priority: 200
+  service:
+    name: federation
+    namespace: federation
+  versionPriority: 10
+  caBundle: "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURwekNDQW8rZ0F3SUJBZ0lKQU52WFZ1TkpFYzhXTUEwR0NTcUdTSWIzRFFFQkN3VUFNR294Q3pBSkJnTlYKQkFZVEFuVnVNUXN3Q1FZRFZRUUlEQUp6ZERFS01BZ0dBMVVFQnd3QmJERUtNQWdHQTFVRUNnd0JiekVMTUFrRwpBMVVFQ3d3Q2IzVXhLVEFuQmdOVkJBTU1JR1psWkdWeVlYUnBiMjR0WTJWeWRHbG1hV05oZEdVdFlYVjBhRzl5CmFYUjVNQjRYRFRFNE1EUXlOREEwTVRBeE0xb1hEVEU1TURReU5EQTBNVEF4TTFvd2FqRUxNQWtHQTFVRUJoTUMKZFc0eEN6QUpCZ05WQkFnTUFuTjBNUW93Q0FZRFZRUUhEQUZzTVFvd0NBWURWUVFLREFGdk1Rc3dDUVlEVlFRTApEQUp2ZFRFcE1DY0dBMVVFQXd3Z1ptVmtaWEpoZEdsdmJpMWpaWEowYVdacFkyRjBaUzFoZFhSb2IzSnBkSGt3CmdnRWlNQTBHQ1NxR1NJYjNEUUVCQVFVQUE0SUJEd0F3Z2dFS0FvSUJBUURGME5zR3ZpUWQ0US9oRHcyVTAvNnIKcDRIWFIrN0pCZTNjKzNlb0pVZlRHT21Gd1lkKy96N25kTFFZVjJpbXl6elR6b3Fjc1RvZ1p4ZllzeE9XdjM3WgorUzNtYklDdGYweGlPOGtsTC80a0JQZUtJU1R6WWE3LzNLYitLOTFnSzAwcHlNVzE4R3V2Y0lsNkk2SFR5VGlICit5aFNEeW5PdjRqQldTMUZteDBZblV6T3pZSHVQU0dBTkQ5Q0cyZHFNTFB2a2xIR2wrUTJyanQvTDNZMVN1cEYKV0NON1dlZk0rVjduVHhXSFd3RjdqT3JlTm1Ha0ozaGZvM0VsZm4zWll1Ri9oSFl1Nm5rOVpYZlVYeGZDald1dAp1Vmd2dytJRDN4UU9IeGV0RW5Vem9GbUVCR2U5ajNkVFQ5eHJrK0pQUTVRSzBPSjhWY0kvaGV4aXo5bURMOVlaCkFnTUJBQUdqVURCT01CMEdBMVVkRGdRV0JCVGdoWStZRnJCT1I4L2tHTVlQbEJnQWpjRHJqREFmQmdOVkhTTUUKR0RBV2dCVGdoWStZRnJCT1I4L2tHTVlQbEJnQWpjRHJqREFNQmdOVkhSTUVCVEFEQVFIL01BMEdDU3FHU0liMwpEUUVCQ3dVQUE0SUJBUUN6SHMvRkR6RDZoa3pzRGtpSlAwK29pLzlRSHpUbnI0M29NSGs1VStoMFk4Q3NrTWpSCmNTS01jSDNURXB5VlUrVGh6RGVsUWo1OG9LWmlNMlFxOVYwbGhHWDlYamVmSG5qcmNPZlJzY1lZWUJ3QTM5QWkKZS9DYnRBUmxUQXZ3QnMxMW5xUU8xQ1NiZjU0L1AxKzJzNU0yK3d0cXo4WEVaZDVPd1VvVVJFd0NMc2pQUmdzTwp3VFQ2M293V3U1Znl3Wm5LZUd1TUFaYXlFTEpYaXUzeXhLWlJpc2lCSXdrZ0hJQ0taN3U1Y3FkSmZrZFk0U3ZLCnVRSjRmMTNKZTNqUkhKTG9uYTBzT3EvUm4rZTdSdVQ5TmtqRTdoQkFFeEdnTnh1N1EyOFUwbkM4UkNJeHdHZGkKcGZLeEpDaU5peVBqaXJ6d04vZm1IdWVMTk1EVCtsSVozTUNWCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: federation
+  namespace: federation
+  labels:
+    api: federation
+    apiserver: "true"
+spec:
+  ports:
+  - port: 443
+    protocol: TCP
+    targetPort: 443
+  selector:
+    api: federation
+    apiserver: "true"
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: federation
+  namespace: federation
+  labels:
+    api: federation
+    apiserver: "true"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        api: federation
+        apiserver: "true"
+    spec:
+      containers:
+      - name: apiserver
+        image: shashidharatd/fv2:v5
```

Comment:
Should this stuff be a separate PR?

Reply:
Sorry, my bad. This was not supposed to be checked in. Removed now.